tests/local/utils/test_mock_etl.py

```diff
@@ -37,3 +37,7 @@ def process(self, df: DataFrame) -> DataFrame:
     def test_mocking(self):
         load = MockLoader()
         load.hello.world.foo.bar()
+
+
+if __name__ == "__main__":
```

Comment:
I think this is unneccesary

Reply:
if it inherits pytest and has the prefix *test_* it will be executed. See https://docs.pytest.org/en/7.1.x/how-to/usage.html

Reply:
It is not for the test to run. It is so i can run the script in the debugger.

Reply:
Since it is for local running in the debugger -maybe it should beremoved before merging? 
I have made the same comment before: https://github.com/atc-net/atc-dataplatform/pull/149

Reply:
If it is a good thing to use - then we should consider use this in all tests-scripts? 
I think it is just my lack of understanding of what it does.

Reply:
It is fair, i will remove it.